Jenkinsfile

```
@@ -61,11 +61,32 @@ node('linux') {
 
     if (infra.isTrusted()) {
         stage('Publish') {
-            sh 'mkdir -p updates'
-            sh 'cp target/*.json target/*.html updates'
+            sh '''
+                mkdir -p updates
+                cp target/*.json target/*.html updates
+            '''
             sshagent(['updates-rsync-key']) {
                 sh 'rsync -avz  -e \'ssh -o StrictHostKeyChecking=no\' --exclude=.svn updates/ www-data@updates.jenkins.io:/var/www/updates.jenkins.io/updates/'
             }
+            withCredentials([
+                string(credentialsId: 'updates-jenkins-io-file-share-url-with-sas-token', variable: 'UPDATES_FILE_SHARE_URL'),
+                string(credentialsId: 'aws-access-key-id-updatesjenkinsio', variable: 'AWS_ACCESS_KEY_ID'),
+                string(credentialsId: 'aws-secret-access-key-updatesjenkinsio', variable: 'AWS_SECRET_ACCESS_KEY')
+            ]) {
+                sh '''
+                ls -al updates/
+
+                azcopy sync updates/ "${UPDATES_FILE_SHARE_URL}" --exclude-path '.svn' --recursive=true --delete-destination=true
```

```suggestion
                azcopy sync updates/ "${UPDATES_FILE_SHARE_URL}" --exclude-path '.svn' --recursive=true
```

=> as the existing `rsync` command does not delete files on the remote SSH server